site/_posts/2019-09-27-Velero-v1-1-on-vSphere.md

```diff
@@ -8,7 +8,7 @@ categories: ['kubernetes']
 # Tag should match author to drive author pages
 tags: ['Velero', 'Cormac Hogan', 'how-to']
 ---
-Velero version 1.1 provides support to backup Kubernetes applications deployed on vSphere. This post will provide detailed information on how to install and configure Velero to backup and restore a stateless application (`nginx`) that is running in Kubernetes on vSphere. At this time there is no vSphere plugin for snapshotting stateful applications on vSphere during a Velero backup. In this case, we rely on a third party program called `restic`. However this post does not include an example of how to backup a stateful application. That is available in another tutorial which can be found [here](./2019-20-09-Velero-v1-1-Stateful-Backup-vSphere.md).
+Velero version 1.1 provides support to backup Kubernetes applications deployed on vSphere. This post will provide detailed information on how to install and configure Velero to backup and restore a stateless application (`nginx`) that is running in Kubernetes on vSphere. At this time there is no vSphere plugin for snapshotting stateful applications on vSphere during a Velero backup. In this case, we rely on a third party program called `restic`. However this post does not include an example of how to backup a stateful application. That is available in another tutorial which can be found [here](velero.io/Velero-v1-1-Stateful-Backup-vSphere.md).
```

Comment:
Two things:
- if we publish this post first, the  `here` link will be broken until the post it points to gets published. We could reorder the post publish dates, or remove the link and just say "...a future post", or accept the broken link for a few days.
- the `.md` needs to be dropped from the end of the link URL

Reply:
Done